src/VolumeCache.ts

```diff
@@ -42,24 +42,27 @@ export type CachedVolume = {
   numChannels: number;
 };
 
-const extentIsEqual = (a: CacheEntryExtent, b: CacheEntryExtent): boolean => {
-  return (
-    a.x[0] === b.x[0] &&
-    a.x[1] === b.x[1] &&
-    a.y[0] === b.y[0] &&
-    a.y[1] === b.y[1] &&
-    a.z[0] === b.z[0] &&
-    a.z[1] === b.z[1]
+/** Fill in partially-specified, invalid, or missing `Box3` with reasonable defaults */
+function applyDefaultsToRegion(region: Box3 | undefined, size: Vector3): Box3 {
+  if (!region) {
+    return new Box3(new Vector3(), size.clone());
+  }
+
+  const min = new Vector3(
+    isFinite(region.min.x) ? region.min.x : 0,
+    isFinite(region.min.y) ? region.min.y : 0,
+    isFinite(region.min.z) ? region.min.z : 0
+  );
+  const max = new Vector3(
+    region.max.x > 0 ? region.max.x : size.x,
+    region.max.y > 0 ? region.max.y : size.y,
+    region.max.z > 0 ? region.max.z : size.z
```

Comment:
Should you also be checking for finite-ness here too?

Reply:
The idea here was based on the fact that `Box3` defaults to min = `Infinity` and max = `-Infinity` for all components, to represent an empty/uninitialized box. I figured the > 0 check would capture this case for max, but I might as well make it more robust and easier to understand anyways.